CHANGELOG.md

```diff
@@ -6,29 +6,21 @@ Full Changelog: [v0.1.0-beta.9...v0.1.0-beta.10](https://github.com/Metronome-In
-* **api:** api update ([c959e5c](https://github.com/Metronome-Industries/metronome-go/commit/c959e5c6a2ea1b6bb2aa096c65b96449bf17d85b))
-* **api:** api update ([ba0171c](https://github.com/Metronome-Industries/metronome-go/commit/ba0171ce3373acd3f5a0c0ca9f28fda96f5a1b7f))
-* **api:** api update ([2e728fe](https://github.com/Metronome-Industries/metronome-go/commit/2e728fe262aa17188a99ac7a41d1dab8697ca58e))
-* **api:** api update ([af56da3](https://github.com/Metronome-Industries/metronome-go/commit/af56da3d7231fe7cffb76b2048cf1ee84c947c3a))
-* **api:** api update ([21433c2](https://github.com/Metronome-Industries/metronome-go/commit/21433c2f88155d470180c0f0126351f00e37142e))
-* **api:** api update ([6cc9155](https://github.com/Metronome-Industries/metronome-go/commit/6cc91553bb22a2fe0ad8cb4c30def9e222bf8c3c))
-* **api:** api update ([cfc1d4d](https://github.com/Metronome-Industries/metronome-go/commit/cfc1d4db02ff32e650b52382ec0911987a5e8051))
-* **api:** api update ([3019739](https://github.com/Metronome-Industries/metronome-go/commit/3019739253ba835e81e9c77bf39d4db031a73e9e))
-* **api:** api update ([d18807f](https://github.com/Metronome-Industries/metronome-go/commit/d18807f041b837a1afd9c1fb184ea48e1adfbc89))
-* **api:** api update ([1553750](https://github.com/Metronome-Industries/metronome-go/commit/1553750f458e67088a73c2f04453e49b9b5b8eb5))
-* **api:** api update ([7793347](https://github.com/Metronome-Industries/metronome-go/commit/77933470076683148375ee38bfd939875c411448))
-* **api:** api update ([da9c2ab](https://github.com/Metronome-Industries/metronome-go/commit/da9c2ab4c1f9cebf4fbac5fc01e3be4cd2c5d36d))
+* **api:** Add Event Search API for finding events to match to customers and billable metrics ([25c419f](https://github.com/Metronome-Industries/metronome-go/commit/25c419f98a132a86d3231305d01e05b8283f2be2))
+* **api:** add previewEvents API for generating draft invoices with provided events ([fe1821e](https://github.com/Metronome-Industries/metronome-go/commit/fe1821ef081398fcabf045d49f8118c8d5010a91))
+* **api:** add support for Anrok and Precalculated tax types in payment gateway configuration ([64985e8](https://github.com/Metronome-Industries/metronome-go/commit/64985e8288143e110a600fb44a3662202f94bd20))
+* **api:** add UUID format annotations to credit type IDs ([7697c03](https://github.com/Metronome-Industries/metronome-go/commit/7697c03b0d64bb0059808c7b1856bd4da7f04dac))
+* **api:** add custom credit type support to prepaid balance thresholds ([4c11f2f](https://github.com/Metronome-Industries/metronome-go/commit/4c11f2fdb146d17523fc3997d0745f3c79af2abb))
+* **api:** add contract priority field ([e62737e](https://github.com/Metronome-Industries/metronome-go/commit/e62737eadd941e1718a58f915bd3087715c40771))
+* **api:** remove previewEvents API ([c959e5c](https://github.com/Metronome-Industries/metronome-go/commit/c959e5c6a2ea1b6bb2aa096c65b96449bf17d85b))
```

Comment:
nit: we can remove both `**api:** add previewEvents API for generating draft invoices with provided events` and `**api:** remove previewEvents API`